src/vm/domainfile.cpp

```diff
@@ -3203,6 +3203,9 @@ void GetTimeStampsForNativeImage(CORCOMPILE_VERSION_INFO * pNativeVersionInfo)
 #endif // FEATURE_CORECLR
 }
 
+static constexpr uint32_t CMOVFlags = 0x00008001;
+static constexpr uint32_t SSE2Flags = 0x04000000;
```

Comment:
The compiler we build our TFS mirror with is older (I don't know why...) and doesn't have support for `constexpr`, so this will cause a build break for our internal builds. (The compiler version is 18.10.40116.6 -- an internal VS2013+ build?). You won't see `constexpr` anywhere else in the code.